homeassistant/components/media_player/__init__.py

```diff
@@ -115,6 +117,38 @@
     ATTR_INPUT_SOURCE,
 ]
 
+# Service call validation schemas
+MEDIA_PLAYER_SIMPLE_SCHEMA = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+})
+
+MEDIA_PLAYER_MUTE_VOLUME_SCHEMA = vol.Schema({
```

Comment:
Not perse a comment, more an FYI. You can extend schema's to built on top of one another. So if you want, you could do something like this:

```python
MEDIA_PLAYER_MUTE_VOLUME_SCHEMA = MEDIA_PLAYER_SIMPLE_SCHEMA.extend({
```